framework/source/class/qx/lang/Json.js

```diff
@@ -230,6 +230,11 @@ qx.Bootstrap.define("qx.lang.Json",
 
 /*! JSON v3.2.5 | http://bestiejs.github.io/json3 | Copyright 2012-2013, Kit Cambridge | http://kit.mit-license.org */
 (function (window) {
+  // This polyfill does not work under Rhino because it cannot convert POJO to object (it tries
+  //  to serialise the class)
+  if (qx.core.Environment.get("runtime.name") === "rhino" || qx.core.Environment.get("runtime.name") === undefined)
+    return;
```

Comment:
You are not only changing the behavior for rhino but also for the case the runtime.name is not set ... please elaborate.